src/plugins/discover/public/application/angular/doc_table/components/pager/tool_bar_pager_text.tsx

```diff
@@ -17,14 +17,12 @@ interface Props {
 
 export function ToolBarPagerText({ startItem, endItem, totalItems }: Props) {
   return (
-    <I18nProvider>
```

Comment:
I18nProvider was omitted due to it's already provided higher in `create_doc_table_embeddable.tsx` file